apps/meteor/tests/e2e/page-objects/omnichannel-units.ts

```diff
@@ -0,0 +1,100 @@
+import type { Locator } from '@playwright/test';
+
+import { OmnichannelAdministration } from './omnichannel-administration';
+
+export class OmnichannelUnits extends OmnichannelAdministration {
+	get inputSearch() {
+		return this.page.locator('[placeholder="Search"]');
+	}
+
+	async search(text: string) {
+		await this.inputSearch.fill(text);
+		await this.page.waitForTimeout(500);
```

Comment:
Instead of a timeout, maybe we could improve the table with the appropriate `aria` props, so we don't have to wait, cause it is not guaranteed that in 500ms the data will be available

Do something akin to https://github.com/RocketChat/Rocket.Chat/pull/30580/files